roles/openshift_logging_elasticsearch/vars/main.yml

```diff
@@ -3,6 +3,7 @@ __latest_es_version: "3_6"
 __allowed_es_versions: ["3_5", "3_6", "3_7"]
 __allowed_es_types: ["data-master", "data-client", "master", "client"]
 __es_log_appenders: ['file', 'console']
+__es_revision_history_limit: 0
```

Comment:
is this something we expect customers to set? if so this doesn't belong in `vars/main.yaml` but rather `defaults/main.yaml`